src/components/Data/Data.js

```diff
@@ -20,6 +20,13 @@ const userData = [
     folderName: "AbhaySimonGame",
     projectTitle: "Simon Game",
   },
+  {
```

Comment:
Undo all the changes made to this file. This file should not be commited as mentioned in the guidelines.